Linkta-org/LandingPage
# LINTKA-381: Style all components and pages to prepare for deployment

Style all components preparing for deployment
Configured 3 screen sizes

![image](https://github.com/Linkta-org/landing-page/assets/94409079/758908a8-25a0-46c7-ba75-a6e9accb4162)
![image](https://github.com/Linkta-org/landing-page/assets/94409079/10ce0ae8-3a95-4f2c-8c35-b8103db309a9)
![image](https://github.com/Linkta-org/landing-page/assets/94409079/04d7db5e-bfdf-4fd4-a716-1f31afc9a822)

## Description



Fixes # (issue) LINKTA-381

## Type of change



- [ ] Other 🗒️ **\*\***\_\_**\*\***

## How Has This Been Tested?

To Test all stylings:
1. Go to localhost:3000 - Home Page (3 screen sizes)
2. Fill out the form - Form States/Error
3. Click Join Waiting list - Check EmailVerificationPrompt
4. Go to Email Click Sign in - Check Thank you Page on localhost:3000/thank-you
5. Go to localhost:3000/not-found - not found page
6. Throw an error on /legal/page.tsx by ```throw new Error ('testing') to check error.tsx

## Checklist:



- [ ] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [ ] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.

## Additional Information:
